packages/instrumentor/plugins/functionHooks.ts

```diff
@@ -68,12 +56,13 @@ function applyHooks(
 	if (!functionNode.params.every((p) => babel.isIdentifier(p))) {
 		return false;
 	}
-
 	if (!matchesResult.hasHooks()) {
-		trackedHooks.available.add(functionName);
+		trackedHooks.available.add(packageName, functionName);
```

Comment:
Could we extend `trackedHooks` with a method to add hooks and not access internal fields directly? This seems like an [Indecent Exposure](https://luzkan.github.io/smells/indecent-exposure) code smell.